digits/templates/layout.html

```diff
@@ -77,15 +77,8 @@
                return base;
             }
             return baseName(suggestion.value);
-        }
-    });
-    $(".autocomplete_path").removeAttr("autocomplete");
-    $(".autocomplete_path").on('keyup',function(e){
-      // If path is relative, enforce absolute
-      var isRelative = e.target.value[0] != '/' && e.target.value[0] != '\\';
-      if (e.target.value.length > 0 && isRelative) {
-        e.target.value = "/" + e.target.value;
-      }
+        },
+        minChars: 0
```

Comment:
Interesting... I also changed the minChars count in addition to removing the problematic line for @JVR32. @IsaacYangSLA  Would you be able to check for me if removing `minChars: 0`  fixes the problem for you? Thanks!